.gitignore

```diff
@@ -18,3 +18,7 @@ node_modules/
 # Non-distributable configs.
 phpunit.xml
 .llmsenv
+
```

Comment:
is this temporary?
I mean, as far as I can see we're not generating those maps anymore (I mean, there's no reference to them in the and mins). Why we don't just delete those map files then?

Reply:
When you "watch" scripts via `wp scripts start` it generates source maps that are useful when developing / testing new code. By default the `assets/` dir would be cleaned by `wp scripts` when compiling production assets *but* I've disabled that because we currently have the source and minified versions of many scripts in the assets folder. If we clean the whole directory we lose most of our scripts.

So my thinking was to .gitignore them and exclude them from the zip archives during distributions that way we don't have to worry about remembering to delete them (or writing a new script to delete them during builds)

Reply:
makes sense. thanks.